src/main/java/codechicken/nei/recipe/RecipeHandlerQuery.java

```diff
@@ -22,6 +22,7 @@
     private final List<T> recipeHandlers;
     private final List<T> serialRecipeHandlers;
     private final String[] errorMessage;
+    private boolean error = false;
```

Comment:
Do I understand correctly that this variable is never reset back to `false`?
And if an error occurs even once then this logic will *always* print error message. Is it intended behavior?

Reply:
New instances of `RecipeHandlerQuery` are created whenever `openRecipeGui` is called, so error state does not persist across queries